src/assets/locales/README.md

```diff
@@ -0,0 +1,10 @@
+# Adding Locales
```

Comment:
Might be good to add a link to this document to the main `README.md` and/or `CONTRIBUTING.md` under a heading about adding translations/locales.

Reply:
Added to `CONTRIBUTING.md`